packages/teams-js/src/public/secondaryBrowser.ts

```diff
@@ -6,14 +6,20 @@ import { ErrorCode } from './interfaces';
 import { runtime } from './runtime';
 
 /**
- * Namespace to power up the in-app browser experiences in the Host App.
- * For e.g., opening a URL in the Host App inside a browser
+ * Namespace to power up the in-app browser experiences in the host app.
+ * For e.g., opening a URL in the host app inside a browser
+ * Namespace to power up the in-app browser experiences in the host app.
+ * For e.g., opening a URL in the host app inside a browser
```

Comment:
Maybe the diff is just showing me something funny here but is this text listed twice here?